mpu6050/mpu6050_data.h

```diff
@@ -0,0 +1,52 @@
+#ifndef __MPU6050_DATA_H__
+#define __MPU6050_DATA_H__
+
+#include <linux/list.h>
+#include <linux/slab.h>
+#include <linux/module.h>
+
+
+enum mpu6050_data_index {
+	INDEX_ACCEL_X = 0,
+	INDEX_ACCEL_Y,
+	INDEX_ACCEL_Z,
+	INDEX_GYRO_X,
+	INDEX_GYRO_Y,
+	INDEX_GYRO_Z,
+	INDEX_TEMPERATURE,
+	INDEX_COUNT,
+	/* Another level of data*/
+	INDEX_TIMESTAMP = 0,
+	INDEX_EXTRA_COUNT
+};
+
+struct mpu6050_data_elements {
+	int data[INDEX_COUNT];
+	u64 extra_data[INDEX_EXTRA_COUNT];
+};
+
+struct mpu6050_data_list {
+	struct list_head list;
+	struct mpu6050_data_elements data;
+};
+
+struct mpu6050_data_holder {
+	struct i2c_client *drv_client;
+	int (*read_data)(bool debug);
```

Comment:
As I can see,
it doesn't make any sense to store the `read_data` function in the `mpu6050_data_holder` structure - it's never been used by `mpu6050_data` module.

Reply:
Done.